sig-scalability/slos/slos.md

```diff
@@ -89,6 +89,8 @@ In order to meet SLOs, you have to use extensibility features "wisely".
 The more precise formulation is to-be-defined, but this includes things like:
 - webhooks have to provide high availability and low latency
 - CRDs and CRs have to be kept within thresholds
+- Applications running in the cluster should use Informers where applicable (rather than making repeated 
```

Comment:
This is good recommendation, but it doesn't belong here.
Our SLOs should be satisifed no matter how controllers are implemented (e.g. via proper load shedding).

I suggest moving this around and adding a section to the FAQ page:
https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/faq.md

Reply:
>Our SLOs should be satisifed no matter how controllers are implemented (e.g. via proper load shedding).

Ah good point.  I can see how that's totally correct for load that is bursty. But what if the baseline load (i.e. average steady state load) on a cluster is higher than what the cluster can cope with? Load shedding will never let it cope with that.  Instead, there will litterally be a bunch of requests that can never be satisfied.

For instance, this PR was inspired by observing a cluster with many different instances of an app, and _each instance_ of that app was listing pods 20 times per minute.  The load was steady and continuous.  Given that the app was written in the way it was, there's no way that load shedding could ever let that app run successfully.  Because the app was written in such a way that each instance needed it's 20 pod listings per minute.  IMHO that particular, coded as it was, simply cannot be made to run on K8s with any realistic-sized master nodes.  The coding of the app _has_ to be changed, if they want it to successfully operate.  That's the point I was trying to convey in this PR..

I can see why this recommendation would fit well into the FAQ page.  But I'm concerned that saying nothing about this here in slo.md can create the impression that it's OK to run continuous workloads of constant large list operations.  And, once you reach a certain combination of list size and request refrequency... that's actually not OK.

So what about putting the details in the FAQ, and then saying something like this here in the SLO page: "If you add software to your cluster which interacts with the API server, be aware that some requests put much more load on the API server than others. LIST requests can be particularly expensive in terms of performance.  The FAQ<link> describes how you should minimize the number of expensive requests."

What do you think wojtek-t?

Reply:
(I just edited the above to improve proposed wording)